.github/ISSUE_TEMPLATE/bug_report.md

```diff
@@ -54,7 +54,7 @@ PASTE HERE OR PROVIDE LINK TO https://pastebin.com/ OR SIMILAR
 
 <details><summary>System Details</summary>
 
-- OS (with version, e.g Windows 10 v2004 or macOS 10.15 (Catalina)):
+- OS (with version, e.g. Windows 10 v2004 or macOS 10.15 (Catalina)):
```

Comment:
```suggestion
- OS (with version, e.g., Windows 10 v2004 or macOS 10.15 (Catalina)):
```

Reply:
Putting a comma in front of e.g. seems weird to me. [The examples here](https://dictionary.cambridge.org/dictionary/english/eg) don't add a comma.

Reply:
AE vs BE: https://jakubmarian.com/comma-after-i-e-and-e-g/